spec/mailers/activity_mailer_spec.rb

```diff
@@ -1,9 +1,11 @@
 require 'rails_helper'
 
 RSpec.describe ActivityMailer, type: :mailer do
-  let(:template_path) { File.join(__dir__, '../fixtures/activity_mailer/test_email') }
+  let(:template_path) { '../../spec/fixtures/activity_mailer/test_email' }
```

Comment:
Thanks! But I've tried this. It doesn't work. Did you mean `mailer` is an instance of `ActivityMailer`? But when I call `new` method, it raises an error "private method `new' called for ActivityMailer:Class"